src/librbd/Group.h

```diff
@@ -0,0 +1,19 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef CEPH_LIBRBD_GROUP_H
+#define CEPH_LIBRBD_GROUP_H
+
+namespace librbd {
+  // Consistency groups functions
```

Comment:
Minor: no indentation required for namespace -- just leave a blank line between the namespace block and its contents for clarity.